tests/codegen/test-project2/object/link_prop_async_edgeql.py.assert

```diff
@@ -0,0 +1,62 @@
+# AUTOGENERATED FROM 'object/link_prop.edgeql' WITH:
+#     $ edgedb-py
+
+
+from __future__ import annotations
+import dataclasses
+import datetime
+import edgedb
+import typing
+import uuid
+
+
+class NoPydanticValidation:
+    @classmethod
+    def __get_validators__(cls):
+        from pydantic.dataclasses import dataclass as pydantic_dataclass
+        pydantic_dataclass(cls)
+        cls.__pydantic_model__.__get_validators__ = lambda: []
+        return []
+
+
+@dataclasses.dataclass
+class LinkPropResult(NoPydanticValidation):
+    id: uuid.UUID
+    name: str
+    friends: typing.List[LinkPropResultFriendsItem]
+
+
+@dataclasses.dataclass
+class LinkPropResultFriendsItem(NoPydanticValidation):
+    id: uuid.UUID
+    name: str
+
+    def __getitem__(self, key: str) -> typing.Any:
```

Comment:
I thought we'd generate proper typed overloads here?

Reply:
Right, this specific `assert` is for Python 3.7 which doesn't have `typing.Literal`

Reply:
Ah, ok, i figured it out -- there's another properly typed .assert